build.sbt

```diff
@@ -11,10 +12,6 @@ lazy val core = project
     name := "http4s-jdk-http-client",
     libraryDependencies ++= coreDeps,
     mimaBinaryIssueFilters ++= Seq(
-      // package private, due to #641
-      ProblemFilters.exclude[IncompatibleMethTypeProblem](
-        "org.http4s.jdkhttpclient.JdkHttpClient.defaultHttpClient"
-      )
```

Comment:
Why don't we need this MiMa filter in main? (I trust the MiMa check more than my brain and since it's not yelling this seems fine!, I just don't understand)

Reply:
MiMA in main does not make sense. We have no binary compatibility guarantees here.

Since the last merge was from 0.10, this was also removed there, since we were already starting a new series.

Reply:
Ahh I see, thank you!